### PR TITLE
Conformance of prior versions

### LINKED ISSUE #544: Do we specify that in the version sequence there must be a monotonic sequence of non-decreasing OCFL version numbers?
Do we specify that in an objects version sequence there must be a monotonic sequence of non-decreasing OCFL version numbers? Ie. something like `v1` is OCFL v1.0, `v2` is OCFL v1.1, `v3` is OCFL v1.1

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #544

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -913,6 +913,17 @@ <h2>Version Inventory and Inventory Digest</h2>
             Non-normative note: Storing an inventory for every version provides redundancy for this critical
             information in a way that is compatible with storage strategies that have immutable version directories.
         </blockquote>
+        <section id="conformance-of-prior-versions">
+          <h2>Conformance of prior versions</h2>
+          <p>
+              Version directories in OCFL are intended to be immutable in that existing version directories do not
+              change when new version directory is added. Each version directory within an OCFL Object
```
Comment (awoods) [inline draft/spec/index.html:920]: typo?
```
..when [a] new version directory..
```

File: draft/spec/index.html [hunk 2]
```diff
@@ -913,6 +913,17 @@ <h2>Version Inventory and Inventory Digest</h2>
...
+              <span id="E103">MUST</span> conform to either the same or a later OCFL specification version as the
```
Comment (awoods) [inline draft/spec/index.html:921]: Is there an `E103`?

**Review by awoods — CHANGES_REQUESTED** [review]

File: draft/spec/index.html (hunk 1 above)
Comment (zimeon) [reply draft/spec/index.html]: Thanks, fixed

File: draft/spec/index.html (hunk 2 above)
Comment (zimeon) [reply draft/spec/index.html:921]: It is new. That reminds me we need to talk about versioning the validation codes file: https://github.com/OCFL/spec/issues/553

**Review by rosy1280 — APPROVED** [review]

File: draft/spec/index.html (hunk 2 above)
Comment (awoods) [reply draft/spec/index.html:921]: I am not seeing a pull-request nor a direct change to [validation-codes.md](https://github.com/OCFL/spec/blob/main/validation/validation-codes.md) with the addition of `E103`. When you say, "It is new", are you saying, "We need to create `E103` in 'validation-codes.md'"? ...or should I be looking somewhere else?

Comment (zimeon) [reply draft/spec/index.html:921]: Yes, we need to add it when #553 is resolved

**Review by neilsjefferies — DISMISSED** [review]

File: draft/spec/index.html (hunk 2 above)
Comment (zimeon) [reply draft/spec/index.html:931]: Addressed with f0161002cae2c397c255e7ca19d22633018f37f6

**Comment by zimeon** [comment]
Commit f0161002cae2c397c255e7ca19d22633018f37f6 not only adds the `E103` error code but also fixes the links in `draft/spec/validation-codes.md` to go to the draft spec and not to 1.0

**Review by rosy1280 — APPROVED** [review]

**Review by awoods — APPROVED** [review]

**Review by neilsjefferies — APPROVED** [review]

**Review by julianmorley — APPROVED** [review]

**Pull request merged**